xml/System.Runtime.Loader/AssemblyLoadContext.xml

```diff
@@ -405,8 +501,20 @@
         <ReturnType>System.Func&lt;System.Runtime.Loader.AssemblyLoadContext,System.Reflection.AssemblyName,System.Reflection.Assembly&gt;</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
-        <remarks>To be added.</remarks>
+        <summary>Occurs when the resolution of an assembly fails.</summary>
+        <remarks>
+          <format type="text/markdown"><![CDATA[
+
+## Remarks
```

Comment:
Mention that this event can be handled even for default ALC.
The LoadUnmanagedDll() and Load() methods are pre-defined for the default and cannot be overridden.